src/find-options/FindOptionsUtils.ts

```diff
@@ -1,9 +1,9 @@
-import {FindManyOptions} from "./FindManyOptions";
-import {FindOneOptions} from "./FindOneOptions";
-import {SelectQueryBuilder} from "../query-builder/SelectQueryBuilder";
-import {FindRelationsNotFoundError} from "../error/FindRelationsNotFoundError";
-import {EntityMetadata} from "../metadata/EntityMetadata";
-import {DriverUtils} from "../driver/DriverUtils";
+import { FindManyOptions } from "./FindManyOptions";
```

Comment:
Please disable prettier in your editor for this project. Someday we'll format the whole codebase using prettier (I hope once we merge all staled PRs), then you can enable it. For now please make it simpler for us to review your PRs.